app/views/videos/index.html.haml

```diff
@@ -0,0 +1,8 @@
+%article.video_category
+  - @categories.each do |cat|

```

Comment:
Tiny nitpick nut you could change the variable name to something like `category` instead if `cat`